sdk-features/redact-sensitive-data.md

```diff
@@ -180,6 +180,34 @@ import { Redacted } from 'cobrowse-sdk-react-native';
 </View>
 ```
 
+**Redaction of Jetpack Compose UI**
+
+Redaction for Jetpack Compose UI is shipped in a separate library on Maven Central:
+
+```
+dependencies {
+    // ... other dependencies ...
+    implementation 'io.cobrowse:cobrowse-sdk-android:2.+'
+    implementation 'io.cobrowse:cobrowse-sdk-android-compose-ui:2.+'
+}
+```
+
+{% hint style="info" %}
+You are supposed to use the same version of the Cobrowse.io SDK and Compose UI redaction archifacts. Using different versions of Cobrowse.io SDK artifacts is not supported.
```

Comment:
typo: archifacts -> artifacts

Reply:
Maybe also change "You are supposed to use " to "You are required to use "

Reply:
Adjusted.